### PR TITLE
[12.0] [MIG] product_firmware_version

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Product firmware version
========================

This module allows you to store the latest/official/tested firmware version on an electronic product.

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@OCA/product-maintainers could you please review code?

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza @bodedra Could you please review it?

File: product_firmware_version/README.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (tbaden) [inline product_firmware_version/README.rst:1]: please use the new README by fragment system and generate the README.rst with the [maintainer-tools](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
code lgtm
just a minor detail

**Comment by Nikul-Chaudhary** [comment]
@tbaden @bodedra Code you please review again?

**Review by bodedra — APPROVED** [review]
Tested on Runbot. LGTM :+1: 

@Nikul-Chaudhary Thanks for migration.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza @rafaelbn Can you please review and move forward.

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza @rafaelbn @max3903 if you don't mind, can you please review it?

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Pull request merged**